docs/source/actions/sns-action.rst

```diff
@@ -0,0 +1,52 @@
+SNS action
+==========
+
+This action allows you to send custom notifications to an AWS `Simple Notification Service (SNS)`_ topic when your monitor suites finish their execution.
+
+To use this action, you need to provide the AWS credentials and the SNS topic ARN in your ``settings.py`` file as follows:
+
+.. code-block:: python
+
+    # settings.py
+    SPIDERMON_SNS_TOPIC_ARN = '<SNS_TOPIC_ARN>'
+    SPIDERMON_AWS_ACCESS_KEY_ID = '<AWS_ACCESS_KEY>'
```

Comment:
In the future, we may want to refactor all Actions using `SPIDERMON_AWS_ACCESS_KEY_ID` to be able to  define their own `SPIDERMON_{ACTION}_AWS_ACCESS_KEY_ID` or to use `SPIDERMON_AWS_ACCESS_KEY_ID` as fallback.

That would help in case you want to use different keys for each type of service. 

cc. @Gallaecio , @fcanobrash  @curita

Reply:
For such cases, users should probably be encouraged to create their own action classes and set those parameters through `super().__init__()`. I imagine there could be scenarios where you want to have 2 instances of the same action with different settings, so you might need to do that anyway.

But I guess an action-specific setting that uses the global one as fallback would indeed minimize scenarios requiring subclassing, so +1 from me.

Reply:
In my opinion we should keep having only one place with AWS credentials to keep the possible settings available manageable. It is more likely that someone will have the same keys for any AWS integration.

`SPIDERMON_AWS_ACCESS_KEY_ID` and `SPIDERMON_AWS_SECRET_ACCESS_KEY` are common keys that any AWS integration will require, and in my experience, it is very unlikely that they will have different ones for different services (and if so, they can create their own custom actions)

I would change `SPIDERMON_SNS_TOPIC_ARN` to `SPIDERMON_AWS_SNS_TOPIC_ARN` to make it more clear that we are setting a AWS setting to the project.

This follows the patter we already have for `SPIDERMON_AWS_REGION_NAME` and `SPIDERMON_AWS_RETURN_PATH`